### ISSUE TITLE
[Bug] The app sometimes crashes on boot, Version: 1.1.0

### ISSUE THREAD

**Issue by Teliang** [IllusionMan1212/lyrics-grabbr#17]
```
08-06 12:53:38.941 23465 23465 E AndroidRuntime: FATAL EXCEPTION: main
08-06 12:53:38.941 23465 23465 E AndroidRuntime: Process: com.illusionman1212.lyricsgrabbr, PID: 23465
08-06 12:53:38.941 23465 23465 E AndroidRuntime: k8.b: Unexpected JSON token at offset 0: Expected start of the object '{', but had 'e' instead at path: $
08-06 12:53:38.941 23465 23465 E AndroidRuntime: JSON input: error code: 1033
08-06 12:53:38.941 23465 23465 E AndroidRuntime: 	at g8.g.n(SourceFile:1)
08-06 12:53:38.941 23465 23465 E AndroidRuntime: 	at g8.g.m(SourceFile:1)
08-06 12:53:38.941 23465 23465 E AndroidRuntime: 	at k8.g.l(SourceFile:57)
08-06 12:53:38.941 23465 23465 E AndroidRuntime: 	at k8.g.m(SourceFile:1)
08-06 12:53:38.941 23465 23465 E AndroidRuntime: 	at k8.g.n(SourceFile:67)
08-06 12:53:38.941 23465 23465 E AndroidRuntime: 	at k8.g.w(SourceFile:48)
08-06 12:53:38.941 23465 23465 E AndroidRuntime: 	at k8.g.f(SourceFile:1)
08-06 12:53:38.941 23465 23465 E AndroidRuntime: 	at k8.f.o(SourceFile:123)
08-06 12:53:38.941 23465 23465 E AndroidRuntime: 	at t6.a.e(SourceFile:8)
08-06 12:53:38.941 23465 23465 E AndroidRuntime: 	at t6.a.b(SourceFile:1)
08-06 12:53:38.941 23465 23465 E AndroidRuntime: 	at k8.f.q(SourceFile:1)
08-06 12:53:38.941 23465 23465 E AndroidRuntime: 	at j8.b.a(SourceFile:31)
08-06 12:53:38.941 23465 23465 E AndroidRuntime: 	at t6.f.a(SourceFile:204)
08-06 12:53:38.941 23465 23465 E AndroidRuntime: 	at t6.e.s(SourceFile:1)
08-06 12:53:38.941 23465 23465 E AndroidRuntime: 	at g7.a.k(SourceFile:1)
08-06 12:53:38.941 23465 23465 E AndroidRuntime: 	at x7.f0.run(SourceFile:1)
08-06 12:53:38.941 23465 23465 E AndroidRuntime: 	at m.j.run(SourceFile:14)
08-06 12:53:38.941 23465 23465 E AndroidRuntime: 	at d8.i.run(SourceFile:1)
08-06 12:53:38.941 23465 23465 E AndroidRuntime: 	at d8.a.run(SourceFile:92)
08-06 12:53:38.941 23465 23465 E AndroidRuntime: 	Suppressed: c8.f: [p1{Cancelling}@f08fd05, Dispatchers.Main.immediate]
08-06 12:53:38.943  1639 16540 W ActivityTaskManager:   Force finishing activity com.illusionman1212.lyricsgrabbr/.MainActivity
08-06 12:53:38.944  1639 23513 I DropBoxManagerService: add tag=data_app_crash isTagEnabled=true flags=0x2
08-06 12:53:38.946  1639  1782 I ActivityManager: Showing crash dialog for package com.illusionman1212.lyricsgrabbr u0
```

**Comment by IllusionMan1212** [comment]
Can you share the name of the song and artist you were listening to when this happened? Makes it much easier to reproduce and fix.

**Comment by Teliang** [comment]
Sorry, I forgot what song I was listening to. I used to listen to Japanese songs. If I reproduce the same situation next time, I will remember the song.

**Comment by Teliang** [comment]
Song: しるし
Artist: Mr.Children

08-16 12:59:19.819  7575  7575 E AndroidRuntime: FATAL EXCEPTION: main
08-16 12:59:19.819  7575  7575 E AndroidRuntime: Process: com.illusionman1212.lyricsgrabbr, PID: 7575
08-16 12:59:19.819  7575  7575 E AndroidRuntime: k8.b: Expected start of the object '{', but had 'EOF' instead at path: $
08-16 12:59:19.819  7575  7575 E AndroidRuntime: JSON input: 
08-16 12:59:19.819  7575  7575 E AndroidRuntime: 	at g8.g.n(SourceFile:1)
08-16 12:59:19.819  7575  7575 E AndroidRuntime: 	at g8.g.m(SourceFile:1)
08-16 12:59:19.819  7575  7575 E AndroidRuntime: 	at k8.g.l(SourceFile:57)
08-16 12:59:19.819  7575  7575 E AndroidRuntime: 	at k8.g.m(SourceFile:1)
08-16 12:59:19.819  7575  7575 E AndroidRuntime: 	at k8.g.n(SourceFile:67)
08-16 12:59:19.819  7575  7575 E AndroidRuntime: 	at k8.g.w(SourceFile:48)
08-16 12:59:19.819  7575  7575 E AndroidRuntime: 	at k8.g.f(SourceFile:1)
08-16 12:59:19.819  7575  7575 E AndroidRuntime: 	at k8.f.o(SourceFile:123)
08-16 12:59:19.819  7575  7575 E AndroidRuntime: 	at t6.a.e(SourceFile:8)
08-16 12:59:19.819  7575  7575 E AndroidRuntime: 	at t6.a.b(SourceFile:1)
08-16 12:59:19.819  7575  7575 E AndroidRuntime: 	at k8.f.q(SourceFile:1)
08-16 12:59:19.819  7575  7575 E AndroidRuntime: 	at j8.b.a(SourceFile:31)
08-16 12:59:19.819  7575  7575 E AndroidRuntime: 	at t6.f.a(SourceFile:204)
08-16 12:59:19.819  7575  7575 E AndroidRuntime: 	at t6.e.s(SourceFile:1)
08-16 12:59:19.819  7575  7575 E AndroidRuntime: 	at g7.a.k(SourceFile:1)
08-16 12:59:19.819  7575  7575 E AndroidRuntime: 	at x7.f0.run(SourceFile:1)
08-16 12:59:19.819  7575  7575 E AndroidRuntime: 	at m.j.run(SourceFile:14)
08-16 12:59:19.819  7575  7575 E AndroidRuntime: 	at d8.i.run(SourceFile:1)
08-16 12:59:19.819  7575  7575 E AndroidRuntime: 	at d8.a.run(SourceFile:92)
08-16 12:59:19.819  7575  7575 E AndroidRuntime: 	Suppressed: c8.f: [p1{Cancelling}@4fcc91c, Dispatchers.Main.immediate]
08-16 12:59:19.819  2547  2583 D LocalImageResolver: Couldn't use ImageDecoder for drawable, falling back to non-resized load.
08-16 12:59:19.820  1658 19360 W ActivityTaskManager:   Force finishing activity com.illusionman1212.lyricsgrabbr/.MainActivity

**Comment by Teliang** [comment]
> Song: しるし Artist: Mr.Children

I play this song again, but the app work well, so weird

**Comment by IllusionMan1212** [comment]
I will take a look at fixing this bug this coming week.

**Comment by IllusionMan1212** [comment]
Fixed by: https://github.com/IllusionMan1212/lyrics-grabbr/commit/8bbd3bd52b2368f0eb120c84fdd0bff17c0107fd
Fix should be available in version 1.1.1 on f-droid in approximately 3-6 days
